ItemRackOptions/ItemRackOptions.lua

```diff
@@ -406,6 +406,7 @@ function ItemRackOpt.SaveSet()
 	-- set.equip[18] = nil
 	ItemRackOpt.ReconcileSetBindings()
 	ItemRackOpt.ValidateSetButtons()
+	ItemRack:FireItemRackEvent("SET_SAVED", setname)
```

Comment:
Please call the event in a way that its name is ItemRack specific, like ITEMRACK_SET_SAVED